docs/unified-test-documentation/dasharo-performance/405-fan-control.md

```diff
@@ -0,0 +1,165 @@
+# Dasharo Performance: Fan Control
```

Comment:
@PrzemyslawBanasiak13 please add new position to `mkdocs` file. Without this the test suite will remain hidden.